cmd/issue_note.go

```diff
@@ -28,6 +28,8 @@ func init() {
 	issueNoteCmd.Flags().Bool("quote", false, "quote note in reply")
 	issueNoteCmd.Flags().Bool("resolve", false, "[unused in issue note command]")
 	issueNoteCmd.Flags().MarkHidden("resolve")
+	issueNoteCmd.Flags().StringP("commit", "", "", "[unused in issue note command]")
+	issueNoteCmd.Flags().MarkHidden("commit")
```

Comment:
The problem is that noteRunFn() will fail to compile because commit variable is undefined.  So I must define the function in issue_note.go.

Reply:
@prarit I've faced the same issue when I added L38, would love to find an alternative to that tho.
Maybe we can create a parent class definition for `issue` and another for `mr`?

Reply:
Ahh.. `noteRunFn` is being used in both `mrNote` and `issueNote`. I see!
Instead of creating a parent definition for both, we could maybe create a more granular function for the common tasks there? So the flag variables could, at least, be populated by each command `Run:` separated.
But ok, lets not bother with it now. With the explanation I think it's fine.

Reply:
@zampierilucas, not a bad idea.  I've been thinking about alternatives as well.  I'll look into it -- no promises though :)